Lib/extractor/formats/opentype.py

```diff
@@ -1046,3 +1049,44 @@ def _extractOpenTypeKerningFromKern(source):
         # there are no minimum values.
         kerning.update(subtable.kernTable)
     return kerning
+
+# -----
+# Color
+# -----
+
+COLOR_LAYERS_KEY = "com.github.googlei18n.ufo2ft.colorLayers"
```

Comment:
this `COLOR_LAYERS_KEY` is unused, because you're using the more high-level alternative approach of `COLOR_LAYER_MAPPING_KEY` combined with "Explode Color Layer Glyphs" filter.
Maybe you could it made optional whether to create multiple layers containing glyphs with the same colorID (i.e. what you're currently doing) vs. creating a single default layer that contains all the glyphs, both the base color glyphs and their "glyphlets", with `colorLayers` key defining the mapping with the respective palette index.

Reply:
Sure, but where should that option be passed in?